arch/arm/boot/dts/bcm2838.dtsi

```diff
@@ -35,6 +35,7 @@
 				<0x40042000 0x2000>,
 				<0x40044000 0x2000>,
 				<0x40046000 0x2000>;
+			interrupts = <1 9 0xf04>;
```

Comment:
Wouldn't that be better as `<GIC_PPI 9 (GIC_CPU_MASK_SIMPLE(4) | IRQ_TYPE_LEVEL_HIGH)>`?

Reply:
The kernel doc apparently hasn't been updated with helper macros:
Documentation/devicetree/bindings/interrupt-controller/arm,gic.txt

Will add.